coursemapper-kg/concept-map/src/pipelines/expand_material.py

```
@@ -108,16 +116,26 @@ def expand_material_graph(self, graph: Graph, push_log_message: Callable):
 
         # Add category nodes
         for category_node_id, category_title, category_embedding, category_weight in category_df[['node_id', 'title', 'embedding', 'weight']].iter_rows():
-            category_node = Node(category_node_id, category_title.replace('Category:', ''), '', 'category', category_weight, f'https://en.wikipedia.org/wiki/{category_title.replace(" ", "_")}', '', False, category_embedding)
+            category_node = Node(category_node_id, category_title.replace('Category:', ''), '', 'category', category_weight, f'https://en.wikipedia.org/wiki/{category_title.replace(" ", "_")}', '', False,False,False, category_embedding)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)